Review section for ComboBox control. (#610)
telerik/maui-docs: controls/combobox/selection.md

---
title: Selection
page_title: .NET MAUI ComboBox Documentation - Selection

> description: Learn more about the selection functionality in Telerik UI for .NET MAUI ComboBox control.

Reviewer: missing definite article

position: 6
slug: combobox-selection
---

# .NET MAUI ComboBox Selection

> ComboBox for .NET MAUI enables the app users to quickly select item/items from the dropdown list. This topic will go through the provided by the ComboBox API related to item/items selection.

Reviewer: ```suggestion
The ComboBox for .NET MAUI component lets app users select items from the dropdown list quickly. This topic explains the ComboBox API related to item selection.
```
